dist/api_data.py

```diff
@@ -1204,6 +1220,21 @@ def __cmp__(self, other):
 
 'WT_CONNECTION.open_session' : Method(session_config),
 
+'WT_CONNECTION.query_timestamp' : Method([
+    Config('get', 'all_committed', r'''
+        specify which timestamp to query: \c all_committed returns the largest
+        timestamp such that all earlier timestamps have committed.  See @ref
+        transaction_timestamps''',
+        choices=['all_committed']),
+]),
+
+'WT_CONNECTION.set_timestamp' : Method([
+    Config('oldest_timestamp', '', r'''
+        future commits and queries will be no earlier than the specified
+        timestamp. Supplied values must be monotonically increasing.
```

Comment:
Timestamps have to be mapped into an integral space (which is really the same thing as saying they compare lexicographically).  It's okay (and not surprising) if the same timestamp is specified multiple times, so I think "monotonically increasing" is accurate.